bootstrap/sql/com.mysql.cj.jdbc.Driver/v001__create_db_connection_info.sql

```diff
@@ -391,6 +391,7 @@ CREATE TABLE IF NOT EXISTS webhook_entity (
     id VARCHAR(36) GENERATED ALWAYS AS (json ->> '$.id') STORED NOT NULL,
     name VARCHAR(256) GENERATED ALWAYS AS (json ->> '$.name') NOT NULL,
     deleted BOOLEAN GENERATED ALWAYS AS (json -> '$.deleted'),
+    status VARCHAR(256) GENERATED ALWAYS AS (json ->> '$.status') NOT NULL,
```

Comment:
I think instead of making changes on v001 , we should use the file v002__create_db_connection_info.sql , with alter command to add column to the webhook_entity  table , and let's make it for mysql and Postgres both